coev/membership_triple.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2025 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package coev
```

Comment:
@7908837174 Can you move MembershipTriple triple, to CoMID Package, follow the CORIM Spe.

This will be a useful addition to CoMID Triples